pkg/wsman/cim/chassis/decoder.go

```diff
@@ -0,0 +1,198 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2023
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+
+package chassis
+
+const CIM_Chassis string = "CIM_Chassis"
+
+// ChassisPackageType values
+const (
+	Unknown ChassisPackageType = iota
+	Other
+	SMBIOSReserved
+	Desktop
+	LowProfileDesktop
+	PizzaBox
+	MiniTower
+	Tower
+	Portable
+	LapTop
+	Notebook
+	HandHeld
+	DockingStation
+	AllInOne
+	SubNotebook
+	SpaceSaving
+	LunchBox
+	MainSystemChassis
+	ExpansionChassis
+	SubChassis
+	BusExpansionChassis
+	PeripheralChassis
+	StorageChassis
+	SealedCasePC
```

Comment:
SMBIOS Reserved 25 ,  SMBIOS Reserved 23 are missing before and after SealedCasePC

Reply:
fixed